tests/fe/handlers_test.py

```diff
@@ -865,3 +866,38 @@ def test_group_request_cancelled(session, users, groups, http_client, base_url):
 
     request = Request.get(session, requester_id=user.id, requesting_id=group.id)
     assert request.status == "cancelled"
+
+
+@pytest.mark.gen_test
+def test_request_logging(session, users, http_client, base_url):  # noqa: F811
+    """Test that the fe request handlers properly log stats"""
+    # It does not appear possible to add a mock plugin to the global plugins here and expect it to
+    # be called by the request handler: the server side will create its own plugin proxy
```

Comment:
Same here, inject the `app` fixture and reference `app.plugins`.

Reply:
Oh, actually, looking at this again, that won't work because `plugins` is initialized in `initialize()` and is per-handler rather than per-app. You're going to have to instead call `get_plugin_proxy()` to get the plugin proxy and then inject the new plugin (but that should work).